Remotion/ObjectBinding/Web/UI/Controls/Validation/IValidationFailureMatcher.cs

```diff
@@ -0,0 +1,30 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+using System;
+using System.Collections.Generic;
+using Remotion.ObjectBinding.Validation;
+
+namespace Remotion.ObjectBinding.Web.UI.Controls.Validation
+{
+  public interface IValidationFailureMatcher
```

Comment:
please check, all the types that don't reference web control could be moved to objectbindingCore/Validation folder/namespace.

Reply:
There are some things where only the xml docs would stop me from moving it to core/validation. Should I still move them or should I leave them be?

Reply:
Also, given an abstract baseclass -> it seems wrong to put the child classes in web while the base class is in core.